Map operator.playground/Contents.swift

```diff
@@ -0,0 +1,74 @@
+import Combine
+import Foundation
+import PlaygroundSupport
+
+extension Publishers {
+    struct CustomMap<Upstream: Publisher, T>: Publisher {
```

Comment:
Is it a test and learn operator right? because it looks not different from the regular map.

Reply:
Yes, just an home made `map`! 😄